pkg/validation/internal/good_practices.go

```diff
@@ -126,3 +129,18 @@ func getUniqueValues(array []string) []string {
 	}
 	return result
 }
+
+// validateCrdDescrptions ensures that all CRDs defined in the bundle have non-empty descriptions.
+func validateCrdDescriptions(crds operatorsv1alpha1.CustomResourceDefinitions) []error {
+	f := func(crds []operatorsv1alpha1.CRDDescription, relation string) []error {
+		errors := make([]error, 0, len(crds))
+		for _, crd := range crds {
+			if crd.Description == "" {
+				errors = append(errors, fmt.Errorf("%s CRD %q has an empty description", relation, crd.Name))
+			}
+		}
+		return errors
+	}
+
+	return append(f(crds.Owned, "owned"), f(crds.Required, "required")...)
+}
```

Comment:
We also need to add a test for this one in: https://github.com/operator-framework/api/blob/master/pkg/validation/internal/good_practices_test.go

Reply:
See, e.g; https://github.com/operator-framework/api/blob/master/pkg/validation/internal/good_practices_test.go

Reply:
feel free to close sorted out.